Makefile

```diff
@@ -43,6 +43,8 @@ install-devtools: build-subdir-devtools install-subdir-devtools $(TOOL_CRATES:%=
 
 uninstall-devtools: uninstall-subdir-devtools $(TOOL_CRATES:%=uninstall-devtool-%)
 
+tools-devtools: tools-subdir-devtools
```

Comment:
Nit: Do these new rules need to be defined as `.PHONY`?

Reply:
Done